ihatemoney/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -84,7 +84,7 @@ msgstr ""
 "choisir un autre"
 
 msgid "Which is a real currency: Euro or Petro dollar?"
-msgstr "Euro ou Petrodollar ?"
+msgstr "Quelle la vraie monnaie : Euro ou Petrodollar ?"
```

Comment:
The correct french is "Quelle est la vraie monnaie"

Reply:
:facepalm: 
Thanks reminding (kindly) that I shouldn't do 2 or 3 things at the same time.

Reply:
Sorry if it was rude, didn't mean to be. I was doing several things at the same time as well

Reply:
It wasn't!